framework/test/unit/modules/dpos_v2/module.spec.ts

```diff
@@ -1051,6 +1047,8 @@ describe('DPoS module', () => {
 					if (delegateAddress.equals(generatorAddress)) {
 						expect(currentDelegate.consecutiveMissedBlocks).toBe(0);
 						expect(currentDelegate.lastGeneratedHeight).toBe(nextForgedHeight);
+					} else if (delegateAddress.equals(delegateAddresses[0])) {
```

Comment:
why do we need to handle this separately? the tests says all the delegates must have missedBlocks 1

Reply:
It was a misunderstanding, I omitted start and end slots thinking a round, but startSlot is previous round's last block, so corrected now. Please check last commit.